frontend-react/src/pages/home/Sections/LiveMapSection.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "../../../global-modules.scss" as *;
+
+.legend {
+    list-style: none;
+
+    li {
+        @include u-float(left);
+        @include u-margin-right(1);
+    }
+
+    span {
+        @include u-border(0);
+        @include u-float(left);
+        @include u-width(1.5);
+        @include u-height(1.5);
+        @include u-margin-top(5px);
```

Comment:
Sorry I should've been clearer with my intent to use utility functions. Please try to match to the nearest USWDS space token (not size literal like a px value). This will allow these measurements to update appropriately in potential future USWDS updates or configuration changes. For reference, see https://designsystem.digital.gov/design-tokens/spacing-units/.